taxi/urls.py

```diff
@@ -1,12 +1,14 @@
 from django.urls import path
 
+from .models import Manufacturer
 from .views import (
     index,
     CarListView,
     CarDetailView,
     DriverListView,
     DriverDetailView,
-    ManufacturerListView,
+    ManufacturerListView, ManufacturerCreateView, CarCreateView, CarUpdateView,
+    ManufacturerUpdateView, ManufacturerDeleteView, CarDeleteView,
```

Comment:
we don't import multiple classes/functions/variables on the same line

Reply:
sry, don't understand how I missed that